src/osal/osal.h

```diff
@@ -40,9 +40,10 @@ enum
 {
   OSAL_TIMEOUT_NOTIMEOUT    = 0,      // return immediately
   OSAL_TIMEOUT_NORMAL       = 10,     // default timeout
-  OSAL_TIMEOUT_WAIT_FOREVER = 0xFFFFFFFFUL
 };
 
+static const uint32_t  OSAL_TIMEOUT_WAIT_FOREVER = 0xFFFFFFFFUL;
```

Comment:
I think it is better to have OSAL_TIMEOUT_WAIT_FOREVER as macro than constant

Reply:
Modified.